core/db/events/model.py

```diff
@@ -65,9 +68,11 @@ class Meta(BaseModel.Meta):
     categories = ListAttribute(default=lambda: [])
     start_date = DateAttribute()
     end_date = DateAttribute()
+    end_date_no_recur = DateAttribute(null=True)
```

Comment:
Long term it makes more sense for this to be included in the recurrence details as this attribute is part of that data model